### ISSUE TITLE
questions about metrics

### ISSUE THREAD

**Issue by chenyang126** [brdav/refign#7]
Dear professor:
Your work is excellent! I have some questions about metrics, after training the cityscapes2acdc , I can just see the whole iou, how can I see each class iou such as road, wall...
best regards

**Comment by brdav** [comment]
Hi,
the IoU class is just a wrapper of the [torchmetrics.JaccardIndex](https://torchmetrics.readthedocs.io/en/v0.9.1/classification/jaccard_index.html) module. So you could set `average="none"` to recover the class-wise IoU.
Best

**Comment by chenyang126** [comment]
Dear professor:
Thanks for your answer! Sorry to disturb you again, but how can I see each loss (loss_src, mixed loss...) after each epoch trained? 
best regards

**Comment by brdav** [comment]
In the config files I provide, a Tensorboard logger is used, see [here](https://github.com/brdav/refign/blob/12bfbf16d478899d8163dd38e8c391436edc1ae9/configs/cityscapes_acdc/refign_hrda_star.yaml#L166).
During/after running the experiments, you can check the logs with`tensorboard --logdir=lightning_logs/`
Alternatively, you can also pick another logger, see the [Pytorch Lightning Docs](https://pytorch-lightning.readthedocs.io/en/1.5.10/common/loggers.html) for options.

**Comment by chenyang126** [comment]
thanks for your help!

**Comment by Jumponthemoon** [comment]
> Hi, the IoU class is just a wrapper of the [torchmetrics.JaccardIndex](https://torchmetrics.readthedocs.io/en/v0.9.1/classification/jaccard_index.html) module. So you could set `average="none"` to recover the class-wise IoU. Best

I change the 'average' parameter to 'none', but still give the mean iou. Don't know what goes wrong.
Would you be more specific to give which line in the code?
